### ISSUE TITLE
Disabling and enabling rules using checkboxes changes their order in CSS

### ISSUE THREAD

**Issue by franciscolourenco** [emmetio/livestyle-sublime-old#145]
When you temporarily disable a rule using the checkboxes in the web inspector,  livestyle deletes the rule and adds it to the end of rules in the css file.

![checkboxes](https://i.imgur.com/XYF2Bxu.gif) --> ![css](https://i.imgur.com/9dlZLnX.gif)

This reveals itself to be a problem when there are order sensitive rules, and your css might get broken just by poking around, and you got no idea why.

Previously, Livestyle would comment rules who got disabled in the web inspector instead of deleting them. It seemed a superior solution because:
1. You didn't have the rule re-ordering problem like now.
2. Non destructive. You wouldn't accidentally delete your css just by disabling / enabling css rules.
3. You could still actually delete rules from the web inspector and they would disappear in the CSS.

The (un)comment behaviour seems a solid metaphor for the web inspector's checkboxes enable/disable action, could we possibly go back to it?
##

**Comment by sergeche** [comment]
LiveStyle never commented out rules. It’s pretty complex behaviour to implement. 
Anyway, I think about it.

**Comment by franciscolourenco** [comment]
Then I was probably referring to when I was using [Tincr](http://tin.cr/) instead.
I don't know how it was implemented but maybe it was saving the modifications chrome does itself when you use the checkboxes:

![https://i.imgur.com/fqXe6bZ.png](https://i.imgur.com/fqXe6bZ.png)

Would it be simpler to just keep the order of the rules without commenting?

Other case not covered, however minor, is when there are rules already commented the code, and chrome presents them in the inspector as disabled. If you enable them in the inspector, nothing happens in the code.

**Comment by sergeche** [comment]
This is Chrome DevTools feature. Anyway, I agree it’s a big issue so I’ll think about it

**Comment by ghost** [comment]
I agree with the OP. LiveScript's behavior in this regard is not what I expect or desire. Cheers.
